src/resources/users/controller.js

```diff
@@ -0,0 +1,28 @@
+const prisma = require("../../utils/database");
+const jwt = require("jsonwebtoken");
+const secretKey = process.env.JWT_SECRET;
+
+async function getProfile(req, res) {
+  const token = req.headers.authorization;
+  console.log({ token });
+
+  jwt.verify(token, secretKey, async (err, decoded) => {
+    if (err) {
+      throw Error("Not Authorized");
+    }
+    console.log("DECODED", decoded);
+
+    try {
+      const profile = await prisma.user.findUnique({
+        where: {
+          id: parseInt(decoded.id),
+        },
+      });
```

Comment:
I'm pretty sure we talked about putting this in a `protect` middleware. What happened?

Reply:
I wasn't understanding how, so I did the easier version and will try to figure out separating them now.